apps/website/src/components/Header.astro

```diff
@@ -86,10 +88,6 @@ const { pathname } = Astro.url;
     gap: var(--space-2);
     align-items: center;
     margin-left: auto;
-
-    @media (max-width: 50em) {
-      display: none;
```

Comment:
why was this removed?

Reply:
This was because on small screen widths, we still want to show the search icon. But with this style, the nav bar does not appear.

![image](https://github.com/iTwin/iTwinUI/assets/45748283/a9b3d224-2a45-437b-aac9-6555250b691b)

That's why I removed this style to get:

![image](https://github.com/iTwin/iTwinUI/assets/45748283/d71ac7d6-5731-4388-a17f-8cf14f1c1c3e)

<!-- ![image](https://github.com/iTwin/iTwinUI/assets/45748283/f8955f82-4351-444f-aeaf-c68b54d4171a) -->